.github/README.md

```diff
@@ -36,7 +21,15 @@ The Connected Spaces Platform provides developers across a range of languages an
 
 It comes in the form of an open-source client-side library, effectively encoding (in binary form) the standards by which applications can interoperably communicate with one another. Standards that we can all contribute to.
 
----
+## 📖 Learn
+
+The official documentation for CSP can be found [here](https://builds.magnoboard.com/connected-spaces-platform/index.html).
+
+* The [Learn](https://builds.magnoboard.com/connected-spaces-platform/learn/learn.html) section includes a range of modules that explain topics ranging from the basic tenets of CSP through to more advanced usage.
+
+* The [Manual](https://builds.magnoboard.com/connected-spaces-platform/manual/manual.html) covers quick start guides, tutorials, plus how to build and debug CSP locally.
+
+* It also includes up-to-date [API Documentation](https://builds.magnoboard.com/connected-spaces-platform/api/library_root.html), which can also be found inline in the CSP public API source.
```

Comment:
As this is a bulleted list, "It" could refer to multiple subjects, would recommend keeping each point unique and refer directly to the target for clarity, however the context can be gathered from the remaining statement. 

I am happy to leave this to your discretion.

Reply:
Good call. Updated!